.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,52 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: 'The name of the tarantool build artifact'
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: 'Clone the expirationd module'
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/expirationd
+
+      - name: 'Download the tarantool build artifact'
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+      # TODO(ylobankov): Just in case remove tarantool that could be previously
+      # installed on the runner. Relevant to self-hosted runners. Remove this
+      # step when only GitHub-hosted runners are used for integration testing.
+      - run: sudo apt-get -y purge 'tarantool*'
+      - name: 'Install tarantool'
+        # TODO(ylobankov): Install package dependencies. Now we're lucky: all
+        # dependencies are already there.
+        run: sudo dpkg -i tarantool*.deb
+
+      - name: 'Install expirationd'
+        run: tarantoolctl rocks make
+
+      - name: 'Install test requirements'
+        run: tarantoolctl rocks install luatest 0.5.6
+
+      - name: 'Run tests'
+        run: .rocks/bin/luatest -v -b
+
+      # TODO(ylobankov): Relevant to self-hosted runners. Remove this step
+      # when only GitHub-hosted runners are used for integration testing.
+      - if: always()
+        run: rm -rf .rocks
+
+      # TODO(ylobankov): Relevant to self-hosted runners. Remove this step
+      # when only GitHub-hosted runners are used for integration testing.
+      - if: always()
+        run: sudo apt-get -y purge 'tarantool*'
```

Comment:
The same as above - let's specify package names instead of mask.

Reply:
I hope this will disappear in this PR due to https://github.com/tarantool/tarantool/pull/6537.

Reply:
Not relevant now.